### PR TITLE
Flexbox changes, updating contrib.

### PR DESCRIPTION
**Pull request by tomgenoni**

### CODE REVIEW

**Comment by tomgenoni** [comment]
@danoc do you think the mixins should be its own file, maybe even trumps? Or should we keep as is? I think it's probably okay as is.

File: src/core/partials/components/_layout.scss [hunk 1]
```diff
@@ -1,107 +0,0 @@
...
-
-  &--scroll-x {
-    overflow-x: auto;
-  }
-
-  &--scroll-y {
-    overflow-y: auto;
-  }
-
-  &--empty {
```
Comment (danoc) [inline src/core/partials/components/_layout.scss]: I think this one is helpful and would love if we kept it in some form.

Comment (tomgenoni) [inline src/core/partials/components/_layout.scss]: How about we create a partials just for centering classes/techniques? There are at least three I can think of, each with their own pros/cons.

Comment (danoc) [inline src/core/partials/components/_layout.scss]: Yeah, I think that's good. :+1:

**Comment by tomgenoni** [comment]
@danoc @kwalker3690 any corrections/concerns?

**Comment by danoc** [comment]
LGTM as long as we explore a partial for centering classes.

**Comment by tomgenoni** [comment]
Yup. https://github.com/optimizely/lego/issues/116

File: src/core/partials/components/_layout.scss (hunk 1 above)
Comment (danoc) [inline src/core/partials/components/_layout.scss]: Turns out this is only used twice in the Optimizely app.

**Pull request merged**